docs/publish/host-an-office-add-in-on-microsoft-azure.md

```diff
@@ -1,225 +1,180 @@
     
-    The web project contains Office Add-in website files so this is the project that you publish to Azure.
-    
-3. In  **Publish Web**, choose  **Import**. 
-    
-4. In  **Import Publish Settings**, choose  **Browse**, and then browse to the place where you saved your publish profile earlier in this topic. Choose  **OK** to import your profile.
-    
-5. In  **Publish Web**, on the  **Connection** tab, accept the defaults and choose **Next**. 
-    
-    Choose  **Next ** again to accept the default settings.
-    
-6. On the  **Preview** tab, choose **Start Preview**. The preview shows you all the files in the web project that will be published to the Azure website.
-    
-7. Choose  **Publish**. Visual Studio publishes the web project for your Office Add-in to your Azure Web Site. 
-    
-8. When Visual Studio finishes publishing the web project, your browser opens and shows a webpage with the text "This web app has been successfully created." This is the current default page for the website.
-    
-    To see the webpage for your add-in, change the URL to use https: and add the path of your add-in's default HTML page. For example, the changed URL should look like https://YourDomain.azurewebsites.net/Addin/Home/Home.html. This confirms that your add-in's website is now hosted on Azure. Copy this URL because you'll need it when you edit the add-in manifest file later in this topic.
+2. Right-click the web project and then choose **Publish**. The web project contains Office Add-in web app files so this is the project that you publish to Azure.
     
+3. On the **Publish** tab:
 
-## Step 6: Edit the add-in manifest file to point to the Office Add-in on Azure
+      - Choose **Microsoft Azure App Service**.
+      
+      - Choose **Select Existing**.
 
+      - Choose **Publish**. 
 
+6. In the **App Service** dialog box, find and choose the web app that you created in [Step 3: Create a web app in Azure](../publish/host-an-office-add-in-on-microsoft-azure.md#step-3-create-a-web-app-in-azure) and then choose **OK**. 
 
-1. In Visual Studio with the sample Office Add-in open in  **Solution Explorer**, expand the solution so that both projects show.
+    Visual Studio publishes the web project for your Office Add-in to your Azure web app. When Visual Studio finishes publishing the web project, your browser opens and shows a webpage with the text "Your App Service app has been created." This is the current default page for the web app.
+
+7. To see the webpage for your add-in, change the URL so that it uses HTTPS and specifies the path of your add-in's HTML page (for example: https://YourDomain.azurewebsites.net/Home.html). This confirms that your add-in's web app is now hosted on Azure. Copy the root URL (for example: https://YourDomain.azurewebsites.net); you'll need it when you edit the add-in manifest file later in this walkthrough.
```

Comment:
Maybe just say "article" instead of "walkthrough", since we aren't technically branding this as a walkthrough.